specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2016-12-01/backupManagement.json

```diff
@@ -1107,101 +1104,6 @@
         "deprecated": false
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupJobs": {
```

Comment:
Autorest does support multiple API versions however today not all SDKs make them available (this is slowly changing).
Regardless there appears to be other breaking changes here as well, e.g. when I diff backupManagement.json and jobs.json I see a lot of removed operations.  What happened to them?